### ISSUE TITLE
Haumea oregen

### ISSUE THREAD

**Issue by Cinobi** [GTNewHorizons/NewHorizons#1083]
Find alot of black plutonium in Haumea , look like the table is lying ( https://docs.google.com/spreadsheets/d/1sRroNCV_4JpxrgKNivrOnSi-vz_-TX-a1-ll6WhwBJs/edit#gid=1348574161 )

**Comment by Dream-Master** [comment]
Need to check it but Haumea is Level 7 or 8 too ?

**Comment by Cinobi** [comment]
it's level 7

**Comment by Dream-Master** [comment]
So no exploit.

**Comment by Cinobi** [comment]
yes, but i mine alot in it - in the hope of netherstar ore, and now i cry alot :D

**Comment by Dream-Master** [comment]
Oh come on when you can travel to Tier 7 Planets you can kill Withers like cows :) Not need Netherstar Ore :)

**Comment by Cinobi** [comment]
mob limit removed?)

**Comment by Cinobi** [comment]
checked pluto and makemake, it all have black pluto init too. Look like Haumea nether star ore just replased by black pluto ore. It's bad :c

**Comment by Dream-Master** [comment]
will check it and change it if the config is wrong.

**Comment by Dream-Master** [comment]
Mix 28 Netherstar ore
B:GalaxySpace_Haumea_false=true

Black plutonium was on too.

**Comment by Dream-Master** [comment]
s i remove black plutonium and we will see if Netherstar ore can be found then wen reset the planet next update.
